java/vector/src/main/java/org/apache/arrow/vector/complex/NullableMapVector.java

```diff
@@ -60,9 +60,6 @@ public static NullableMapVector empty(String name, BufferAllocator allocator) {
 
   private final List<BufferBacked> innerVectors;
 
-  private final Accessor accessor;
-  private final Mutator mutator;
-
   // deprecated, use FieldType or static constructor instead
   @Deprecated
   public NullableMapVector(String name, BufferAllocator allocator, CallBack callBack) {
```

Comment:
Do we want to keep both `MapVector` and `NullableMapVector`?

Reply:
Let's discuss when we are removing all non-nullable vectors.

Reply:
track in ARROW-1710